src/app/share/views/SharedLinksView/components/SharedItemList.tsx

```diff
@@ -171,9 +171,11 @@ const ShareItemNameField = ({ shareItem, onItemDoubleClicked, onNameClicked }) =
     <div className={'flex h-full w-full flex-row items-center space-x-4 overflow-hidden'}>
       <div className="relative flex h-10 w-10 shrink items-center justify-center">
         <Icon className="flex h-full justify-center drop-shadow-soft" />
-        <div className="absolute -bottom-0.5 -right-1.5 flex h-5 w-5 items-center justify-center rounded-full bg-primary text-white ring-2 ring-surface">
-          <img src={usersIcon} width={13} alt="shared users" />
-        </div>
+        {shareItem.dateShared && (
```

Comment:
Why does this icon only is displayed when the item has `dateShared?`

Reply:
@CandelR 
I left a note about it in the PR description (**Additional Notes**). Let me know if it's okay or if I should remove the change.